backend/fms_core/serializers.py

```diff
@@ -275,11 +275,9 @@ class Meta:
         depth = 1
 
 class ProjectSerializer(serializers.ModelSerializer):
-    samples = serializers.PrimaryKeyRelatedField(read_only=True, many=True)
     class Meta:
         model = Project
-        fields = "__all__"
-        extra_fields = ('samples')
+        fields = ("id", "name", "principal_investigator", "requestor_name", "requestor_email", "targeted_end_date", "status")
```

Comment:
I'm curious as to why you changed the fields = "__all__" to explicitely listing all the fields from the Project model?

Reply:
Because I think actually the extra fields were superfluous (cause "samples" is included in __alll__)

Reply:
so I need to explicitly list them to avoid __samples__ to get serialized

Reply:
It is a Model Serializer ... samples are not part of the Project model so it should not appear in _all_ i think ...